integration/ejb/perftest/scripts/eapcmp.sh

```diff
@@ -151,9 +151,9 @@ function update_ear {
   [ $? = 0 ] || fatal "perftest dir doesn't exist"
 
 #  if [ ! -f perf-eap5/harness/ear/target/ear-1.0.ear ]; then
-#    mvn clean install -f perf-api/pom.xml
-#    mvn clean install -f perf-eap6/pom.xml
-    mvn clean install
+#    ../../../build.sh clean install -f perf-api/pom.xml
+#    ../../../build.sh clean install -f perf-eap6/pom.xml
+    ../../../build.sh clean install
```

Comment:
We do require that the WORKSPACE env variable points to the directory where build.sh resides so using that would be less brittle.